pycbc/filter/matchedfilter.py

```diff
@@ -1148,15 +1148,15 @@ def get_cutoff_indices(flow, fhigh, df, N):
     kmax: int
     """
     if flow:
-        kmin = int(flow / df)
+        kmin = int(numpy.ceil(flow / df))
```

Comment:
@WuShichao I think changing this could be somewhat dangerous in a lot of places. I'm not sure of all the impacts, so I'm a bit reticent to change this here.

Reply:
@ahnitz Yes, this may have some impacts on other parts of PyCBC, but I think we need to use the same `kmin` in both places, otherwise, it will be divided by 0 in the `sigma` function (because `get_cutoff_indices` use a smaller `kmin`, and `read.py` uses `np.zeros` to initialize the psd.)

Reply:
@WuShichao I agree it should be the same, but the original convention is used everywhere, so I don't think we want to change that.

Reply:
@ahnitz Maybe we can initialize the PSD with `np.inf` instead of `np.zeros`.